Source/VolunteerReporting/Web/Features/CaseReportReceptionFeatures/TextMessageContentParserTests.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
I assume we'll switch to XUnit or some other test framework later to avoid this dependency

Reply:
Agreed!